atlasdb-api/src/test/java/com/palantir/atlasdb/config/LeaderConfigTest.java

```diff
@@ -63,7 +63,7 @@ public void lockLeaderDefaultsToBeTheFirstLeader() {
     public void cannotCreateALeaderConfigWithNoLeaders() {
         ImmutableLeaderConfig.builder()
                 .localServer("me")
-                .leaders(emptySet())
+                .leaders(Collections.emptySet())
```

Comment:
I'd prefer that we are consistent about what we use and at the moment we use both static and non-static versions. I've been bitten a few times where people have statically imported `copyOf` from `ImmutableSet` and I've accidentally lost the ordering of a `List`. Some I find are OK (and baseline excludes them by default) like `assertThat`, `verify`, `mock`, etc.

Thoughts?